smartseq2_single_sample/README.md

```diff
@@ -0,0 +1,62 @@
+- Align reads of each sample to reference genome (using STAR)
+- Perform quality control on generated BAM files (using Picard)
+- Count reads in features (using FeatureCounts)
+- Count reads in features (using RSEM)
+
+# Download
+Use git clone `git clone git@github.com:HumanCellAtlas/skylab.git`
+# Requirements
+## Dockers
+- STAR(2_5_3a) docker: `humancellatlas/star_dev:v1`
+- Picard(2.10.10) docker: `humancellatlas/picard`
+- RSEM docker: `humancellatlas/rsem`
+## Genome Reference
+Generated genome index files using instructions, seen in [STAR Manual](https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf) and [Build RSEM index](http://deweylab.biostat.wisc.edu/rsem/rsem-prepare-reference.html). The genome indexes and refernece files saved to google buckets and need only be generated once for each genome/annotaitons combinations. Besides to the genome indexes, you need:
+- refFlat
+- genome fasta
+- interval list of rRNA
+
+# Example Data
+The example data are curated from [Patel 2014](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4123637/). Samples SRA ids are listed below:
+- SRR1294492 
+- SRR1294493 
+- SRR1294494 
+- SRR1294495 
+- SRR1294496 
+- SRR1294497 
+- SRR1294498 
+- SRR1294499 
+- SRR1294500 
+- SRR1294501 
+- SRR1294876 
+- SRR1295068 
+- SRR1295077 
+- SRR1295078 
+- SRR1295079 
+- SRR1295080 
+- SRR1295081 
+- SRR1295082 
+- SRR1295083 
+- SRR1295084 
+- SRR1295085 
+- SRR1295086
+
+# Output Description
+## STAR Alignment
+RNA-Seq reads are aligned to reference genome using STAR with `--twopass Basic` option. Two bam files are produced by STAR, `Aligned.sortedByCoord.out.bam` and `Aligned.toTranscriptome.out.bam`  
+## Reads counting
+Counting sequencing reads in features(gene/exon/transcript) is done with [FeatureCounts](http://bioinf.wehi.edu.au/featureCounts/) using uniquely mapped reads. The raw counts table contains the Ensembl (gene/transcript/exon) IDs (rows) and the raw read counts per library (columns)
+## Expression quantification
+Gene/isoform expression estimation is done with [RSEM](http://deweylab.biostat.wisc.edu/rsem/rsem-calculate-expression.html) with the `Aligned.toTranscriptome.out.bam` as `--bam` input. The expresison table contains Ensembl gene IDs, transcripts IDs and effective length of gene/isoform(in bp)(rows) and the `expected_count`,`TMP` and `FPKM` per library/cell (columns). `Expected_Counts` of gene is subtracted specifically from gene expression results. 
+
+## Quality control metrics
+Two quality control metrics are done with Picard, `CollectAlignmentSummaryMetrics` and `CollectRnaSeqMetrics`. 
+# Limitations
+1. Running STAR with default parameters
+	1. max number mismatches: Default value of maximum number of mismatches for a pair of reads (paired-end) is 10. The value shoule be dependent on the read length
+	2. max multimapping: Default value of maximum multiple mapping location is 10. 
+2.  No action on checking Fastq quality
+# Contact
```

Comment:
Add contact email? Eventually could be some form of generic help email but for now could be author email?